packages/browser/src/index.js

```diff
@@ -23,20 +24,51 @@ import * as constants from './constants';
 // shared theme
 import colors from './components/theme/colors';
 
+import './antd.css';
+
 // shared store
 const store = configureStore();
 
+function WithConfigProvider(props) {
+	return (
+		<ConfigProvider prefixCls="dejavu-browser">
+			{/* eslint-disable-next-line react/prop-types */}
+			{props.children}
+		</ConfigProvider>
+	);
+}
+
+const Flex = () => (
+	<WithConfigProvider>
+		<DefaultFlex />
+	</WithConfigProvider>
+);
+
+const FlashMessage = () => (
+	<WithConfigProvider>
+		<DefaultFlashMessage />
+	</WithConfigProvider>
+);
+
+const ConnectApp = () => (
+	<WithConfigProvider>
+		<DefaultConnectApp />
+	</WithConfigProvider>
+);
+
 const DataBrowserWrapper = props => (
-	<Provider store={store}>
-		<BrowserRouter>
-			<section>
-				<FlashMessage />
-				<ConnectApp {...props} />
-				{/* eslint-disable-next-line react/prop-types */}
-				<DataBrowserContainer hasCloneApp={props.hasCloneApp} />
-			</section>
-		</BrowserRouter>
-	</Provider>
+	<WithConfigProvider>
+		<Provider store={store}>
+			<BrowserRouter>
+				<section>
+					<FlashMessage />
+					<ConnectApp {...props} />
```

Comment:
@ShahAnuj2610 you can use `<DefaultConnectApp {...props} />` here. as you are already wrapping with config provider